apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/README.md

```diff
@@ -0,0 +1,13 @@
+# Nux Welcome Tour Modal
+
+A help tour to show new users some of the basics of using the editor.
+
+### Testing Instructions
+
+Enable by adding this line to your `0-sandbox.php`
+
+```
+    define('SHOW_WELCOME_TOUR', true);
```

Comment:
Thanks a lot for thinking of this. I agree we should have a README with some basic information about what the feature does. 👍 

The testing instructions, assuming we'll eventually turn this version of the Welcome Tour on for everyone, will be outdated.

I wonder if we could remove the references to `0-sandbox.php` and ` define('SHOW_WELCOME_TOUR', true);`, but keep the line with the line to the Github PR (which will be still useful for testing existing sites).

What do you think @autumnfjeld ?

Reply:
ok, I just took that section out and left the link to the PR with test instructions

Reply:
This is great @roo2 .  +1 for thinking about the little important things like README!